### ISSUE TITLE
Add pronouns attribute to Person

### ISSUE THREAD

**Issue by zichy** [schemaorg/schemaorg#2935]
`Person` should get a new `pronouns` attribute. It may contain one or more gendered or non-gendered pronouns.

Further information:
* https://springfield.edu/gender-pronouns
* https://pronoun.is

### Discussed in https://github.com/schemaorg/schemaorg/discussions/2925

<div type='discussions-op-text'>

<sup>Originally posted by **shaedrich** July 21, 2021</sup>
Hi, I don't know if you already considerd that but besides the [`gender` property](https://schema.org/gender) (which you solved well by defining it as `GenderType|Text`) there's an increasing use of `pronouns`  on the web so what do you think of adding it as a property?</div>

**Comment by shaedrich** [comment]
@zichy Is this issue inspired by this discussion (just asking for linking purposes)
### Discussed in https://github.com/schemaorg/schemaorg/discussions/2925

<div type='discussions-op-text'>

<sup>Originally posted by **shaedrich** July 21, 2021</sup>
Hi, I don't know if you already considerd that but besides the [`gender` property](https://schema.org/gender) (which you solved well by defining it as `GenderType|Text`) there's an increasing use of `pronouns`  on the web so what do you think of adding it as a property?</div>

**Comment by zichy** [comment]
@shaedrich It‘s not inspired by this discussion, but you may link the two. I searched all issues for pronouns, but forgot to have a look into the discussions.

**Comment by shaedrich** [comment]
Okay, cool! So we got the same idea independently&mdash;even better :+1:

I guess, for the link to work, you must add it to the first post of this issue:
```
### Discussed in https://github.com/schemaorg/schemaorg/discussions/2925

<div type='discussions-op-text'>

<sup>Originally posted by **shaedrich** July 21, 2021</sup>
Hi, I don't know if you already considerd that but besides the [`gender` property](https://schema.org/gender) (which you solved well by defining it as `GenderType|Text`) there's an increasing use of `pronouns`  on the web so what do you think of adding it as a property?</div>
```

**Comment by zichy** [comment]
Since this could be a simple fix, does anyone know how long it might take to implement this feature?

**Comment by shaedrich** [comment]
I guess, #1112 has been a similar change. That doesn't look much, does it?

**Comment by zichy** [comment]
It really doesn't.

**Comment by zichy** [comment]
Any updates on this? 🙏

**Comment by shaedrich** [comment]
@zichy Indeed: I just created a PR 🎉

**Comment by shaedrich** [comment]
Although, in languages like German, grammatical gender is different from what is usually perceived as gender, there are living beings, that are not considered "persons" but can have pronouns.

**Comment by TallTed** [comment]
> there are living beings, that are not considered "persons" but can have pronouns

Yes. I intended to mention pets as examples. I usually refer to my pets with pronouns matching their observable physical sex (though I believe that all members of kingdom Animalia have similar chromosomal variance as homo sapiens, and thus my apparently female cats might in fact be mixed-sex, though this *probably* would have been noticed during their surgical spaying)....

Gender is a complex social construct, and crosses numerous boundaries. Perhaps there should be some subproperties or related properties to handle some of these complexities. Perhaps `schema:pronouns` should have a `range` that accommodates this complexity, which might in fact be the very generic `entity` or `resource. I don't have The One True Answer, but I hope readers will at least consider my points against the existing propositions above.

**Comment by danbri** [comment]
@TallTed are you expecting software to do something useful with your pet pronoun metadata if it were expressible as schema.org? Something that it couldn't do if it were in a pets namespace managed alongside? For any area of life we touch upon we can always zoom in and add more and more nuance, but it comes with costs - more complex schemas, more expressivity means more ways descriptions can vary from each other and create harder to use data, etc.

**Comment by TallTed** [comment]
@danbri — 

Assigning `pronouns` a `range` of `person` means that many tools will prevent, or at least error on, use of that property on entities of any type not included in that range.

Perhaps it could at least be changed to the more flexible `rangeIncludes`?

**Comment by danbri** [comment]
AFAIK we have only ever used rangeIncludes here. I don't think many tools
are making a fuss either way. At worst some RDFS tools might assume
Person-hood for pets was implied. But then the definition at /Person says "A
person (alive, dead, undead, or fictional)." so anyone getting too worried
there is probably barking up the wrong tree...

**Comment by TallTed** [comment]
> At worst some RDFS tools might assume Person-hood for pets was implied.

It seems to me that the "at worst" would be assuming *non*-Person-hood for pets. Certainly there are People (humans) who believe their pets, and even animals in general, are also Persons. There are likewise People (humans) who believe that no animals are Persons. Given that what is being constructed here is meant to (eventually) allow machines to operate without human intervention, rigorous construction makes the most sense to me; hence, my commenting at all.

**Comment by fthobe** [comment]
@gmackenz can be closed as DefinedTerm allows 100% flexibility to anyone to implement as desired any value and any consuming platform to implement any value desired into their systems.

https://schema.org/pronouns

**Comment by TallTed** [comment]
Just to make note of it... Many languages assign gender to proper and even improper nouns, and this gender carries through to the pronouns to be used when indirectly referring to those entities. English *usually* doesn't gender non-living entities, but, for instance, ships are commonly referred to with `she` or `her` pronouns. `Pronouns` should therefor never be assigned a `range` of `Person`. `rangeIncludes` is workable.

**Comment by fthobe** [comment]
@TallTed like it works now basically, or am I missing anything?

**Comment by TallTed** [comment]
The [schema:pronouns](https://schema.org/pronouns) page now explicitly says, "A short string listing or describing pronouns for a person ... Used on these types: [schema:Person](https://schema.org/Person)", and all the discussion on that page is focused only on people.

There's a lot of discussion on [*this* page](https://github.com/schemaorg/schemaorg/issues/2935), in [_this_ issue](https://github.com/schemaorg/schemaorg/issues/2935), that I would hope makes clear to all readers that gendered pronouns are *not* used only on schema:Person, but are also used (at least) on ships and pets in English, and on many, many non-Person entities in other languages.

In other words, the [schema:pronouns](https://schema.org/pronouns) page *should* say that it is a "short string listing or describing pronouns for a person *or other entity*" and the other parts of the page should be likewise expanded.

**Comment by fthobe** [comment]
You are technically correct that those are personal pronouns (pronouns aimed at people) and that in grammar also object related pronouns exist. We can adjust the comment, but I don't think it was the scope of the change to add pronouns also to objects as the need was not there. 

`A short string listing or describing pronouns for a person....`

to 

`A short string listing or describing personal pronouns for a person....`

As of today pronouns in schema can not be attributed to objects and I saw no plan to change that.

**Comment by TallTed** [comment]
> As of today pronouns in schema can not be attributed to objects and I saw no plan to change that.

I am saying that such a plan should be made and executed, that failing to do so is an error that will only get harder to rectify.

**Comment by fthobe** [comment]
@TallTed the discussion was about adding pronouns to a person (to cover outcome variability), not about adding them to an object. This is already going further than other ontologies such as wikipedia. Can you provide an example of a mainstream Ontology in which pronouns are listed on Thing or any other equivalent class that has a distinct exclusion of persons and solely applies to objects?

**Comment by TallTed** [comment]
I am not suggesting "a distinct exclusion of persons", nor that `schema:pronoun` should "solely [apply] to objects".

I am suggesting *not* distinctly excluding non-person things, and that `schema:pronoun` SHOULD NOT "solely [apply] to [persons]".

At the moment, it doesn't matter to me what other ontologies have done or are doing on this front. In this issue, it matters what schema.org is doing. Through this issue, schema.org has an opportunity to do the *right* thing, quite possibly stepping beyond what other ontologies are doing. I am saddened to see the direction this discussion is going.

**Comment by danbri** [comment]
Schema.org “Person” is the class of entities we describe as “A person
(alive, dead, undead, or fictional).”

If you can imagine it being somewhat personny, that ticks the “fictional”
box and it simply *is* a schema:Person. Our class means just that. Always
has.

**Comment by TallTed** [comment]
I would not have thought to class a ship, nor a pet, nor a storm, as a "fictional person".

I rather doubt most users of schema.org would think that way.

At the very least, to my mind, the `schema:Person` and `schema:pronoun` pages now cry out for some examples of these sorts.

**Comment by fthobe** [comment]
Man, believe me, I totally get your rift, I go sailing and have a dog. It's not that I do not understand what you want, neither do I call my dog nor a boat "it", for me both are most definitely a "She". Let me break down why I believe that's right now a bad idea and deserves a separate issue.

**Regarding Humans and Animals**

I will surely make, a little bit down the line, a proposal to introduce animals including individualAnimal analogue to [individualProduct](https://schema.org/IndividualProduct)  which would be the natural place to add pronouns on animals if that's what finds support. If you would like to add this to the wish list so that I don't miss it, it's in #746 . I would personally not support it, but neither oppose it and the maintainers will surely make a call on that. Putting it on thing is, I believe, not the right place in regard to pronouns and animals and the target individualAnimal doesn't exist (yet) and if I make PRs I try not make them in a way that needs to be reworked.

**Regarding Objects**
There might be good reasons to give any object from a battery powered Brad Pitt to a train or boat a pronoun, it's not that I don't see your point and nobody stops you to do so (nor to mark it up), schema will validate with custom values, they are just not part of the standard as of right now. I just feel that goes beyond the current scope of schema (and also outside of the original scope of the issue) and if I make a PR (personally believing in it or not) it would get rejected for sure. If you feel we need that, open a ticket, reference this one and it will most probably be discussed. 

Would that work for you so that this ticket can be closed and adding pronouns on animals when they are introduced can be discussed on 746?
